mapadroid/worker/WorkerBase.py

```diff
@@ -1181,3 +1182,41 @@ def _get_screen_size(self):
         self._resocalc.get_x_y_ratio(
             self, self._screen_x, self._screen_y, x_offset, y_offset)
 
+    def _check_data_distance(self, data):
+        mode = self._mapping_manager.routemanager_get_mode(self._routemanager_name)
+        if mode in ["mon_mitm", "iv_mitm"]:
+            data_to_check = "wild_pokemon"
+        else:
+            data_to_check = "forts"
+        lat_sum, lng_sum, counter = 0, 0, 0
+        for cell in data:
+            for element in cell[data_to_check]:
+                counter += 1
+                lat_sum += element["latitude"]
+                lng_sum += element["longitude"]
+        if counter == 0:
+            return None
+        avg_lat = lat_sum / counter
+        avg_lng = lng_sum / counter
+        distance = get_distance_of_two_points_in_meters(float(avg_lat),
+                                                        float(avg_lng),
+                                                        float(self.current_location.lat),
+                                                        float(self.current_location.lng))
+        max_radius = self._mapping_manager.routemanager_get_max_radius(self._routemanager_name)
```

Comment:
routeman get max radius can return None, but this is never checked - should either return a value or check for None

Reply:
I don't think there should be a case where max_radius doesn't exist - but if there were one, the whole check wouldn't make sense, thus I'd just return True in that case.